master/reference/public-cloud/aws.md

```diff
@@ -0,0 +1,115 @@
+does not cover containers running on hosts. Calico implements fine-grain policy
+for containers which dynamically applies to containers as they dynamically
+move around the cluster.
+
+#### No Overlays
+
+Calico minimizes the need for overlays by assigning each container
+its own IP and allowing containers to talk to one another in a pure IP network.
+
+#### Break Through the '50  Route Limit'
+
+[AWS VPC Routing tables are limited to 50 programmable routes](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_Appendix_Limits.html#vpc-limits-route-tables).
+While this might limit other networking plugins, Calico supports
+ip-over-ip encapsulation to circumvent manually programming container routes in
+the VPC routing table altogether.
+
+## How to Run Calico in AWS
+
+Depending on the architecture of your AWS deployment, some of the following
+modifications may need to be made to a stock calico cluster in order for
+it to function properly.
+
+#### Allow BGP Peering in Security Groups
+
+To allow Calico containers to communicate across hosts on AWS, each host
+should have IP connectivity with one another and be able to BGP peer.
+
+To allow this BGP peering, ensure your security group settings for each
+calico host allows incoming tcp connections to port 179.
+
+You can check if your hosts have successfully established BGP sessions with
+one another using `calicoctl node status`.
+
+#### Allow Traffic Within a Single VPC Subnet
+
+By default, each EC2 instance performs source/destination checks on inbound
+and outbound traffic. This ensures that the instance is the source or
+destination of any traffic it sends or receives.
+
+Since Calico allocates a unique IP to each container using pools AWS is not aware
+of, container traffic will have an "unrecognized" source address and will be
+dropped as it passes through the ec2 hosts to the container.
+
+This reverse-path filtering is enabled by default, but can be disabled on each
+host, allowing calico traffic to pass through unaffected.
+**Ensure you follow
+[AWS documentation on disabling source/destination checks](http://docs.aws.amazon.com/AmazonVPC/latest/UserGuide/VPC_NAT_Instance.html#EIP_Disable_SrcDestCheck).**
+
+#### Allow Traffic Across Different Subnets
+
+Container traffic that traverses different subnets will past through
+a VPC routing table. If this VPC routing table does not know how to route
+Calico IPs, it will drop traffic. There are two ways two prevent this from
+happening:
+
+1. Masquerade Container Traffic
+
+   Container traffic can be masqueraded with its host IP so that the VPC
+   never sees the container IPs, allowing standard ec2 routing to take over.
+
+   Turn on traffic masquerading in pool settings by enabling:
+
+   - `ipip` for container-to-container traffic.
+   - `nat-outgoing` for container-to-ec2-instancce traffic.
+
+   See [pool configuration](todo) for information on how to enable this for pools.
```

Comment:
We can at least link to the pool reference guide for now and update when we create a pool configuration guide.